src/components/Form/Form.tsx

```diff
@@ -0,0 +1,86 @@
+import { useState } from 'react';
+import usersFromServer from '../../api/users';
+import { Todo } from '../../types/Todo';
+import { getUserById } from '../../services/user';
+import todosFromServer from '../../api/todos';
```

Comment:
The `todosFromServer` should not be imported here. The logic for calculating the new `id` should be moved to the `App` component, and the `Form` component should receive necessary data via props.